keyboards/moonlander/moonlander.c

```diff
@@ -128,7 +128,20 @@ void moonlander_led_task(void) {
         wait_ms(150);
     }
 #endif
-
+#ifdef CAPS_LOCK_STATUS
+    else {
+        led_t led_state = host_keyboard_led_state();
+        if(led_state.caps_lock) {
+            ML_LED_1(true);
+        }
+        else {
+            uint8_t layer = get_highest_layer(layer_state);
+            if(layer != 1) {
```

Comment:
Hello @fdidron this line is resulting in some unexpected behavior on my board. Can you help me understand why an exception was carved out for layer 1?

Reply:
Oh, because at the time of implementation this was changing LED 1. It now changes the LED for layer 6 though, so this condition should be changed.